src/web/CreateWebComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { memo, forwardRef } from 'react';
+import { WebShape } from './WebShape';
+import { BaseProps } from '../types';
+
+const CreateComponent = memo(forwardRef(WebShape));
+
+export const createComponent = <T,>(
```

Comment:
Is comma in `<T,>` necessary?

Reply:
I guess, it's necessary :)
https://stackoverflow.com/a/45576880